tools/flang2/flang2exe/lldebug.cpp

```diff
@@ -3371,7 +3390,7 @@ lldbg_emit_type(LL_DebugInfo *db, DTYPE dtype, SPTR sptr, int findex,
                     init_subrange_bound(db, &ubv, upper_bnd, 0, findex);
 
                   subscript_mdnode =
-                      lldbg_create_subrange_mdnode(db, lbv, ubv, st);
+                      lldbg_create_subrange_mdnode(db, (LL_MDRef) NULL, lbv, ubv, st);
```

Comment:
Instead of `NULL`, should you pass `ll_get_md_null()` instead?

Reply:
Yes, done in latest. Thanks.